sw/airborne/subsystems/ins/ins_int.c

```diff
@@ -374,6 +378,7 @@ void ins_int_update_gps(struct GpsState *gps_s)
 
 #if INS_USE_GPS_ALT
   vff_update_z_conf(((float)gps_pos_cm_ned.z) / 100.0, INS_VFF_R_GPS);
+  vff_update_vz_conf(((float)gps_speed_cm_s_ned.z) / 100.0, INS_VFF_VZ_R_GPS);
```

Comment:
Maybe it should be possible to enable/disable the velocity update separately from the alt update?
@gautierhattenberger what do you think?

On a sidenote: it might be nice to change these defines to variables, so you can change this via settings during runtime and just use the defines to set the initial value at init...